aws/aws-backend-task.json

```diff
@@ -1,48 +1,36 @@
 {
-    "taskDefinitionArn": "arn:aws:ecs:ap-southeast-1:364680233920:task-definition/kbb-backend-task:1",
+    "family": "kbb-backend-task",
+    "taskRoleArn": "arn:aws:iam::364680233920:role/ecsTaskExecutionRole",
+    "executionRoleArn": "arn:aws:iam::364680233920:role/ecsTaskExecutionRole",
     "containerDefinitions": [
         {
-            "name": "kbb-backend",
-            "image": "public.ecr.aws/f0v2z5x5/kbb-backend",
+            "name": "backend-container",
+            "image": "364680233920.dkr.ecr.ap-southeast-1.amazonaws.com/kbb-backend:latest",
             "cpu": 0,
+            "memory": 1000,
+            "memoryReservation": 500,
             "portMappings": [
                 {
-                    "name": "backend-container-port",
                     "containerPort": 5000,
-                    "hostPort": 5000,
-                    "protocol": "tcp",
-                    "appProtocol": "http"
+                    "hostPort": 0,
```

Comment:
just have a check why hostPort:0

Reply:
I read abit more. 0 is using dynamic port. This document mentioned about dyanamic vs static port. Dynamic port can do multiple instance of the container in the ec2. 

https://docs.aws.amazon.com/AmazonECS/latest/bestpracticesguide/networking-networkmode-bridge.html